src/lib.rs

```diff
@@ -303,6 +303,11 @@ where
                 std::mem::drop(parent_span);
                 span!(HIGHEST_LEVEL, "child").entered()
             };
+
+            // Disable mocking in the child process, fixes test failures on macOS
+            #[cfg(test)]
+            c::mock::disable_mocking();
```

Comment:
It would be nice to know exactly _why_ this fixes tests before merging.

Reply:
I actually don't think this is a prereq to merging this anymore. Some version of this is already in `main` with the merge of #56 and it will be helpful to run MacOS tests from my Linux laptop via this action right away.